samples/android_run_tests/app/build.gradle

```diff
@@ -0,0 +1,51 @@
+apply plugin: 'com.android.application'
+
+android {
+    signingConfigs {
+        Multy_core_config {
+            keyAlias 'key0'
+            keyPassword 'multytest'
+            storeFile file('/Users/pavel/Documents/project/multy/MUL-648/Multy-Core/samples/android_run_tests/Multy_core_tests')
```

Comment:
Please remove the hardcoded absolute path and find a way to make that path relative to project root or something.